### PR TITLE
Feat/add 22 1 0 parity & general updates

### PR DESCRIPTION
**Pull request by Tilo-K**
# This updates the library to match the current PUBG Developer API
I'm planing to use this library for a project and wanted to update some things which are missing.

Changes:
- Added new Telemetry Objects
- Fixed all tests
- Added Masteries & LifetimeStat Service + Tests
- Change the target framework to .NET 8
- updated the README logo
- updated console regions
- added new player attributes

This is a big change. This also breaks compatibility with .netstandard 2.0, but considering this project has been inactive for a while now, I think it's fine to make this change to get up to speed.

If you have any feedback, please let me know.

### CODE REVIEW

File: src/pubg-dotnet/Models/Common/PubgPlatform.cs [hunk 1]
```diff
@@ -9,13 +10,19 @@ public enum PubgPlatform
...
         [EnumMember(Value = "steam")]
         Steam,
+        
+        [Obsolete("The Xbox shard is obsolete use Console instead.")]
         [EnumMember(Value = "xbox")]
         Xbox,
+        
         [EnumMember(Value = "psn")]
+        [Obsolete("The PlayStation shard is obsolete use Console instead.")]
         PlayStation,
```
Comment (GavinPower747) [inline src/pubg-dotnet/Models/Common/PubgPlatform.cs]: With the dotnet update we're going to be incrementing to a new major version anyway so we may as well just remove these

**Review by GavinPower747 — CHANGES_REQUESTED** [review]
You know it's funny enough, I was only thinking about just going back and updating this project a day or two ago. At a cursory glance the code looks fine to me, let's just remove the Xbox and PSN members of the platform enum just since we'll be releasing a new major version anyway I don't think there's a need to mark them as obsolete

**Comment by GavinPower747** [comment]
I'm presuming in the inactivity CI is broken as well which is fair enough so I'll just merge this and get all of that working hopefully over the weekend

**Comment by Tilo-K** [comment]
I removed the deprecated regions. Thank you for the quick initial response and If you need a maintainer going forward, I'd be happy to help out.

File: src/pubg-dotnet/Models/Common/PubgPlatform.cs (hunk 1 above)
Comment (Tilo-K) [reply src/pubg-dotnet/Models/Common/PubgPlatform.cs]: These are removed now.

**Pull request merged**